libcrux-ml-kem/Cargo.toml

```diff
@@ -24,10 +24,9 @@ libcrux-platform = { version = "0.0.2-alpha.3", path = "../sys/platform" }
 libcrux-sha3 = { version = "0.0.2-alpha.3", path = "../libcrux-sha3" }
 libcrux-intrinsics = { version = "0.0.2-alpha.3", path = "../libcrux-intrinsics" }
 
-# This is only required for verification.
-# The hax config is set by the hax toolchain.
-[target.'cfg(hax)'.dependencies]
-hax-lib = { version = "0.1.0-alpha.1", git = "https://github.com/hacspec/hax/" }
+# This is only required for verification, but we are setting it as default until some hax attributes are fixed
+hax-lib = { path = "../../hax/hax-lib" }
```

Comment:
Mmh, `hax-lib` should not be set to a relative path, this will not work, btw the CI says "failed to read `/home/runner/work/libcrux/hax/hax-lib/Cargo.toml`" you need to comment that line out and restore `hax-lib = { git = "https://github.com/hacspec/hax", branch = "main" }`.